### PR TITLE
fix view.add/fork perms and read telepath.yaml in cellauth

### PR DESCRIPTION
**Pull request by invisig0th**

### CODE REVIEW

**Review by Cisphyx — DISMISSED** [review]

File: synapse/cortex.py [hunk 1]
```diff
@@ -2451,6 +2451,10 @@ async def _initCoreViews(self):
             ldef = {'name': 'default'}
             ldef = await self.addLayer(ldef=ldef, nexs=False)
             layriden = ldef.get('iden')
+
+            role = await self.auth.getRoleByName('all')
+            await role.addRule((True, ('layer', 'read')), gateiden=layriden, nexs=False)
```
Comment (vEpiphyte) [inline synapse/cortex.py:2456]: Do we need to retroactively add this permission as well to the ``all`` rule for existing cortexes?

File: synapse/tools/cellauth.py [hunk 2]
```diff
@@ -267,13 +268,21 @@ async def main(argv, outprint=None):
     global outp
     outp = outprint
 
+    fini = None
```
Comment (vEpiphyte) [inline synapse/tools/cellauth.py:271]: could this be cleaner as a ``contextlib.ExitStack`` ?

File: synapse/cortex.py (hunk 1 above)
Comment (invisig0th) [reply synapse/cortex.py:2456]: I debated how to do this, we can discuss

File: synapse/tools/cellauth.py (hunk 2 above)
Comment (invisig0th) [reply synapse/tools/cellauth.py:271]: not sure exactly what that would look like.  lets chat about it.

**Review by vEpiphyte — APPROVED** [review]

**Pull request merged**